### ISSUE TITLE
Support for openapi spec 3.0 ?

### ISSUE THREAD

**Issue by dhirajshrotri** [agreatfool/swagger-spec-to-pdf#18]
I was trying out this library with a swagger JSON file that is in the openapi v3.0 spec format. I noticed that some fields were excluded from the final pdf output. Fields like host and the responseBody object in the path are not being output in the final pdf. Any way these fields can be included in the final output?

**Comment by agreatfool** [comment]
You can try to replace the editor embedded: https://github.com/agreatfool/swagger-spec-to-pdf#swagger-editor. Better to test the displayed (rendered) HTML page by the editor you wanna replace is as your expectation first.

**Comment by dhirajshrotri** [comment]
So for using a new version do I have to replace the files in ./swagger-editor with all the files in the swagger-editor git repo?

**Comment by agreatfool** [comment]
No, you need to find a distribution, a standalone runnable pack. Not the files in the official git repo.
